kusama-guide/sidebars.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
               items: [
                 "learn/learn-treasury",
                 "learn/learn-assets",
-                "general/kusama/kusama-statemine",
```

Comment:
If you are changing these, then you have to implement redirects as well so the links on third party websites are not broken.

Reply:
I'm referring to 

                "learn/learn-staking-guides",
                "learn/learn-identity-guides",
                "learn/learn-assets-guides",
                "learn/learn-vault-guides",